crates/rpc-types-trace/src/parity.rs

```diff
@@ -300,8 +300,23 @@ pub struct CallAction {
     pub value: U256,
 }
 
+/// Creation method.
+#[derive(Clone, Copy, Debug, Default, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "lowercase")]
+pub enum CreationMethod {
+    /// None
+    #[default]
+    None,
```

Comment:
is this actually a valid variant?

can't see any `"none"` in the referenced pr

should this be an Option instead and serialize null?

Reply:
I copied this field from `CallType`, as it also has the `None` variant

```rust
pub enum CallType {
    /// None
    #[default]
    None,
    /// Call
    Call,
```